docs/event/event.md

```diff
@@ -37,7 +37,7 @@ producer.send(producerRecord);
 
 * For cloud-based architectures, evaluate the use of [CloudEvents](https://cloudevents.io/). CloudEvents provide a standardized [Event Envelope](../event/event-envelope.md) that wraps an event, making common event properties such as source, type, time, and ID universally accessible, regardless of how the event itself was serialized.
 
-* In certain scenarios, Events may represent commands (instructions, actions, and so on) that should be carried out by an Event Processor reading the events. See the [Command Event](../event/command-event.md) pattern for details.
+* In certain scenarios, Events may represent commands (instructions, actions, and so on) that should be carried out by an Event Processor reading the events. See the [Command](../event/command.md) pattern for details.
```

Comment:
```suggestion
* In certain scenarios, Events may be interpreted as commands (instructions, actions, and so on) that should be carried out by an Event Processor reading the events. See the [Command](../event/command.md) pattern for details.
```

Reply:
As a best practice, I'd always emit the event as an event, with no intention of treating it as a command. If the consumer wants to interpret it as a command, that's their choice.

Reply:
Yes, I concur with that! Thank you!